src/snorkel_extractor_location/Snorkel_Pipeline_Version_NB3.ipynb

```diff
@@ -858,14 +300,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import json\n",
     "\n",
     "# Setting filename\n",
-    "out_filename = \"loc_ext_test_discriminative.jsonl\"\n",
+    "out_filename = \"phone_ext_test_discriminative.jsonl\"\n",
```

Comment:
would make this `loc_ext_test_discriminative.jsonl`

Reply:
Done